Common/Exceptions/PasswordMissmatchException.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace Common.Exceptions
+{
+    public class PasswordMismatchException : Exception
+    {
+        private const string PasswordMismatchExceptionMessage = "Incorrect password entered";
```

Comment:
"your password and confirmation password do not match"

Reply:
this exception means that old password you entered is incorrect and its not about new password